dipy/viz/tests/test_fvtk_actors.py

```diff
@@ -0,0 +1,211 @@
+import os
+import numpy as np
+
+from dipy.viz import actor, window
+
+import numpy.testing as npt
+from nibabel.tmpdirs import TemporaryDirectory
+from dipy.tracking.streamline import center_streamlines, transform_streamlines
+from dipy.align.tests.test_streamlinear import fornix_streamlines
+
+
+@npt.dec.skipif(not actor.have_vtk)
+@npt.dec.skipif(not actor.have_vtk_colors)
+@npt.dec.skipif(not window.have_imread)
```

Comment:
Since this triplet of tests repeats several times here, maybe you want a single:

`run_test = actor.have_vtk and actor.have_vtk_colors and window.have_imread`

And then `skipif(not run_tests)`?